internal/services/cdn/cdn_frontdoor_firewall_policy_resource_test.go

```diff
@@ -624,6 +844,128 @@ resource "azurerm_cdn_frontdoor_firewall_policy" "test" {
 `, tmp, data.RandomInteger)
 }
 
+func (r CdnFrontDoorFirewallPolicyResource) logScrubbingEnabledBasic(data acceptance.TestData) string {
+	tmp := r.template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_cdn_frontdoor_firewall_policy" "test" {
+  name                = "accTestWAF%d"
+  resource_group_name = azurerm_resource_group.test.name
+  sku_name            = azurerm_cdn_frontdoor_profile.test.sku_name
+  mode                = "Prevention"
+
+  log_scrubbing {
+    enabled = true
+  }
+}
+`, tmp, data.RandomInteger)
```

Comment:
We're pushing away from this pattern as it's unnecessary to assign the template to a var when it can be passed into the test directly.  Would you mind updating these? (Just the new tests, don't worry about the others in the file, we'll be sorting this out more broadly in the future, just want to avoid more instances of it where possible. 🙈 )
```suggestion
	return fmt.Sprintf(`
%s

resource "azurerm_cdn_frontdoor_firewall_policy" "test" {
  name                = "accTestWAF%d"
  resource_group_name = azurerm_resource_group.test.name
  sku_name            = azurerm_cdn_frontdoor_profile.test.sku_name
  mode                = "Prevention"

  log_scrubbing {
    enabled = true
  }
}
`, r.template(data), data.RandomInteger)
```

Reply:
Since I will already be in the code I might as well just update all of them. Fixed.